forward/ft_compute_leadfield.m

```diff
@@ -336,14 +336,24 @@
       %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
       % finite element method as implemented in software duneuro
       %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-
+      
+      % show the license (only once)
+      duneuro_license; 
+      
       %TODO: involve unit checking
 
       % compute secondary leadfield numerically
-      lf = leadfield_duneuro(dippos, headmodel);
+      lf = leadfield_duneuro(dippos, headmodel, 'meg');
 
       % compute primary B-field analytically
-      mu = 4*pi*1e-4; %unit: Tmm/A
+      % check unit of permeability constant mu
```

Comment:
I don't think that unit conversions in the low-level code are appropriate. This code is called very often, e.g. when dipole fitting. Better ensure (once) at t he higher level that the units meet the requirements of being expressed according to SI. Also: if people want to mix units and work with arbitrary units, we don't have to stop them.

Reply:
solved in commit c9080cd. Now the permeability constant mu is in si units.